lib/auth/auth_test.go

```diff
@@ -716,6 +711,28 @@ func (s *AuthSuite) TestCreateAndUpdateUserEventsEmitted(c *C) {
 	c.Assert(s.mockedAuditLog.EmittedEvent.Fields[events.EventUser], Equals, "some-auth-user")
 	s.mockedAuditLog.Reset()
 
+	// test create user with existing user
+	err = s.a.CreateUser(ctx, user)
+	fmt.Println("err: ", err)
+
+	c.Assert(trace.IsAlreadyExists(err), Equals, true)
+	c.Assert(s.mockedAuditLog.EmittedEvent, IsNil)
+
+	// test uninit createdBy gets set to default
+	user2, err := services.NewUser("some-other-user")
+	c.Assert(err, IsNil)
+	err = s.a.CreateUser(ctx, user2)
+	c.Assert(err, IsNil)
+	c.Assert(s.mockedAuditLog.EmittedEvent.Fields[events.EventUser], Equals, "system")
```

Comment:
nit: use `teleport.UserSystem` instead of `"system"`

Reply:
revised